pkg/util/logging/log_counter_hook.go

```diff
@@ -47,7 +51,20 @@ func (h *LogCounterHook) Fire(entry *logrus.Entry) error {
 	defer h.mu.Unlock()
 
 	h.counts[entry.Level]++
-
+	if h.entries[entry.Level] == nil {
+		h.entries[entry.Level] = &persistence.Result{}
+	}
+	ns, isPresent := entry.Data["namespace"]
+	if isPresent {
+		h.entries[entry.Level].Add(ns.(string), errors.New(entry.Message))
```

Comment:
Do you need to check for error field here like below?

Reply:
went through the code, does seem like it's worht adding "error"

Will also additionally add "name" and "resource" to the message